framework/packages/abstract-core/src/native/account_factory.rs

```diff
@@ -111,6 +114,8 @@ pub enum ExecuteMsg {
         description: Option<String>,
         // Account link
         link: Option<String>,
+        /// Account id on the remote chain. Will create a new id (by incrementing), if not specified
+        account_id: Option<AccountId>,
```

Comment:
Correct me if I'm wrong but if we migrate the current version of this contract then sub-account creation from old factory contracts should still work because it won't provide this field which is interpreted as None.

Reply:
Yes, that works !